### PR TITLE
Adding basic documentation

### PR DESCRIPTION
**Pull request by azharichenko**
- Added a new DOCS.md file for API documentation
- Added LICENSE

Added a simple DOCS.md file to address issue #20. The current documentation takes from what is already in the code. I'm planning on further improving the documentation of DOCS.md and the Pitt API especially with possible forcing conformity with [PEP 257](https://www.python.org/dev/peps/pep-0257/).

Also added the LICENSE file since this is under GPLv2  and should include a copy of the license by default.

### CODE REVIEW

**Comment by azharichenko** [comment]
Update:
- Added location dictionaries to API documentation
- Fixing of text in the documentation

File: DOCS.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+### **get_dining_locations_menu(location, date)**
+
+#### **Parameters**:
+  - `location`: Dining location
+  - `date`: Date
+
+#### **Returns**:
+---
+
+### **_encode_dining_location(string)**
```
Comment (RitwikGupta) [inline DOCS.md]: This is a private method so it shouldn't be present in a DOCS.md. Just docstrings in code.

File: DOCS.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+#### **Parameters**:
+  - `string`: Text
+
+#### **Returns**:
+
+---
+
+### **_decode_dining_location(string)**
```
Comment (RitwikGupta) [inline DOCS.md]: Private method, same as above

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (RitwikGupta) [inline LICENSE]: Scratch this, we most likely want MIT.

**Comment by azharichenko** [comment]
Ok, I updated the docs with any reference to private functions removed.

**Comment by RitwikGupta** [comment]
Closes #20

**Pull request merged**